CheckDrive.Api/CheckDrive.Application/Services/Authorization/AuthService.cs

```diff
@@ -0,0 +1,106 @@
+﻿using AutoMapper;
+using CheckDrive.Application.Constants;
+using CheckDrive.Application.DTOs.Identity;
+using CheckDrive.Application.Interfaces.Authorization;
+using CheckDrive.Domain.Exceptions;
+using Microsoft.AspNetCore.Identity;
+
+namespace CheckDrive.Application.Services.Authorization;
+
+public class AuthService : IAuthService
+{
+    private readonly JwtHandler _jwtHandler;
+
+    private readonly UserManager<IdentityUser> _userManager;
+
+    private readonly RoleManager<IdentityRole> _roleManager;
+
+    private readonly IMapper _mapper;
+
+    private readonly IEmployeeRegistrationService _employeeFactory;
```

Comment:
no need for separating them with an empty line. They belong to the same logical code.